### DISCUSSION TITLE
Importing TFS into GitHub

### DISCUSSION THREAD

**Discussion by SSSEric** [community/community#33864] [Other Feature Feedback, Questions, & Ideas]
Our company is looking to move to Github from TFS. I created an organization and then ran the import option for TFS.

![image](https://user-images.githubusercontent.com/114087497/191818956-6938d6f6-b05c-4c7e-bba9-5ea65d7b9a0d.png)

It prompted me for my credentials, which I entered. I am an administrator so I used my account.

It prompted me to select the project that I wanted to import so I selected **Web Projects (tfs)**.

![image](https://user-images.githubusercontent.com/114087497/191819167-7aac6487-bc72-4011-832b-5a41520caada.png)

I then matched the appropriate users to Github's users.

![image](https://user-images.githubusercontent.com/114087497/191819218-931251d0-5f89-4a3a-ab0b-8846ad51c696.png)

I received an email that the import was complete so I logged in and checked the repository.

What I noticed is that it did not import all of the projects. Actually, it only imported one folder, and only the contents of this folder. I have highlighted in red the one it brought in below.

![image](https://user-images.githubusercontent.com/114087497/191819273-fd2c0a93-0ba7-4c97-b961-5f5fb703701f.png)

I've run this import 3 times with the same results. My user account on TFS has full rights on that server to do whatever I need to do.

I've tried to import other folders but didn't get the full import of those either.

Is there something I need to do on the TFS side of things to get this to import all of the project folders within the Web Portal folder?

Any suggestions would be greatly appreciated. We really would like Github to work for us but we need our history and code imported to use it.

Thank you!

**Comment by TechGuyAlabama** [comment]
Anyone have ANY suggestions?
